Cargo.lock

```diff
@@ -5281,7 +5281,6 @@ dependencies = [
  "spin-config",
  "spin-core",
  "spin-key-value",
- "spin-key-value-redis",
```

Comment:
Not sure I understand why this line is being removed here.

Reply:
cc @fibonacci1729 @lann